cmd/dfget/app/root.go

```diff
@@ -299,12 +299,12 @@ func handleNodes() error {
 
 func resultMsg(cfg *config.Config, end time.Time, e *errortypes.DfError) string {
 	if e != nil {
-		return fmt.Sprintf("download FAIL(%d) cost:%.3fs length:%d reason:%d error:%v",
+		return fmt.Sprintf("download FAIL(%d)\ncost:%.3fs length:%d\nreason:%d\nerror:%v",
 			e.Code, end.Sub(cfg.StartTime).Seconds(), cfg.RV.FileLength,
 			cfg.BackSourceReason, e)
 	}
-	return fmt.Sprintf("download SUCCESS(0) cost:%.3fs length:%d reason:%d",
-		end.Sub(cfg.StartTime).Seconds(), cfg.RV.FileLength, cfg.BackSourceReason)
+	return fmt.Sprintf("download SUCCESS !\ncost:%.3fs length:%d",
```

Comment:
output them in one line and don't change the content, `0` is the return code, it represents success.

Reply:
I think success(0) will make others confused, success or failed?

Reply:
success don't need a code to present it is seccess, success is 0, 0 is success.but success(0) make people confused.At least when I first use Dragonfly,I was confused with it

Reply:
I think it is better to be ```
fmt.Sprintf("download SUCCESS cost:%.3fs length:%d reason:%d",
		end.Sub(cfg.StartTime).Seconds(), cfg.RV.FileLength, cfg.BackSourceReason)
```

Reply:
```
--2019-08-22 20:27:58--  https://dl.google.com/go/go1.12.7.src.tar.gz
dfget version:
workspace:4889-1566476878.241
sign:4889-1566476878.241
download FAIL(1104) cost:0.373s length:-1 reason:1001 error:{"Code":1104,"Msg":"download fail and not back source: 1001"}
exit status 80

--2019-08-22 20:28:40--  https://dl.google.com/go/go1.12.7.src.tar.gz
dfget version:
workspace:5126-1566476920.745
sign:5126-1566476920.745
client:127.0.0.1 connected to node:127.0.1.0:8001
start download by dragonfly...
download SUCCESS cost:4.380s length:21976647 reason:0

```